x-pack/test/functional/services/machine_learning/api.ts

```diff
@@ -247,5 +247,16 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
         }
       });
     },
+
+    async createCalendar(calendarId: string, body = { description: '', job_ids: [] }) {
+      await esSupertest
+        .put(`/_ml/calendars/${calendarId}`)
+        .send(body)
+        .expect(200);
+    },
+
+    async deleteCalendar(calendarId: string) {
+      await esSupertest.delete(`/_ml/calendars/${calendarId}`).expect(200);
+    },
```

Comment:
Similar to `createCalendar`: assert that the calendar has really been deleted.

Reply:
Check https://github.com/elastic/kibana/pull/52269/commits/09334634271c5888e38c223250f7b8952bc4e500